pkg/stub/config.go

```diff
@@ -220,7 +219,23 @@ func (h *Handler) buildFileMaps(cfg *v1.Config, forceRebuild bool) error {
 			}
 		}
 	}
-	return nil
+
+	cm, err := h.configmapclientwrapper.Get(util.IST2ImageMap)
+	if kerrors.IsNotFound(err) {
+		cm = &corev1.ConfigMap{}
+		cm.Name = util.IST2ImageMap
+		cm.Namespace = v1.OperatorNamespace
+		cm.Data = map[string]string{}
+		for key, value := range h.imagestreatagToImage {
+			cm.Data[key] = value
+		}
+		_, err = h.configmapclientwrapper.Create(cm)
```

Comment:
Do you see any need for a retry here if we get an error creating it for some reason?

Reply:
I think the propagation of the error to the caller will effectively get us a retry, and it is called out of the main reconciliation loop.

And hence I'd rather avoid added stuff here.